single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -0,0 +1,365 @@
+
+![Architecture](./images/fs_architecture.png)
+As the diagram shows above, running on Microsoft Fabric, a data pipeline lands, ingests and transforms the incoming data. The transformed data is registered as features in Azure Machine Learning Managed Feature Store. The features are then can be used for model training and inferencing. In the meantime, the data lineage of both the data pipeline and features are tracked and monitored by Azure Purview.
+
+## Getting Started
+
+### Environment Setup
+
+#### Prerequisites
+
+- Create an Azure subscription if you haven't got one. You can get a free trial [here](https://azure.microsoft.com/en-us/free/).
+- Define a prefix, such as 'fsd1', which will be used to name the resources created in this demo. Please note that the prefix must be unique within the Azure subscription.
+
+#### Microsoft Fabric
+
+##### Step 1: Create a Microsoft Fabric workspace
+
+After signing into Microsoft Fabric, it's necessary to create a new workspace for setting up a pipeline. Therefore, please navigate to the left-hand menu, select the workspace option, and proceed to create a new workspace.
+
+![workspace_01](./images/data_pipeline/workspace_01.png)
+
+Next, please enter your workspace name and description (optional). If you wish, you can upload a photo to use as your workspace avatar. This is also optional. However, please note that you need to select 'trial' under the 'license mode' in the advanced options, as currently only this type supports all the features of Fabric. Then click the 'Apply' button to complete the creation of the workspace.
+
+![workspace_02](./images/data_pipeline/workspace_02.jpg)
+
+##### Step 2: Create a Lakehouse
+
+Then within this created workspace, create a new Lakehouse named as '[prefix]_lh' by clicking the 'New' button on the top of the page. Please note that [prefix] is the prefix you defined in the prerequisites section.
+
+##### Step 3: Import the sample notebooks
+
+After the Lakehouse is created, please click the 'New' button again, and select 'Import notebook' from the menu. Navigate to `src/notebooks` folder under the repo, and select all the notebooks to import:
+
+![import all notebooks](./images/import_all_notebooks.png)
+
+##### Step 4: Add the Created Lakehouse to the notebooks
+
+After the notebooks are imported, please open the notebook named as 'data_cleansing'，'data_ingestion'，'data_transformation','data_validation'，‘EDA','feature_set_registration' respectively, and add the Lakehouse created in the previous step to the notebook. Please note that the Lakehouse name should be '[prefix]_lh', where [prefix] is the prefix you defined in the prerequisites section.
+
+![add lakehouse](./images/add_lh_to_notebook.png)
+
+#### Create an Azure Resource Group
+
+Create a new resource group named as [prefix]rg, where [prefix] is the prefix you defined in the prerequisites section.
+
+#### Azure App Service
```

Comment:
I think this would require some additional details. I am in the app service and don't have any idea what configuration to choose. Also, please add reference links to the docuemtation.

Reply:
Thanks Anuj! Bo has confirmed that the App Service is not required anymore, we'll remove this in later commits, thanks! :- )

Reply:
Thanks for point out this. i will remove this section and app service code.

Reply:
updated!